src/game/common/globaldata.h

```diff
@@ -63,6 +63,8 @@ class GlobalData : public SubsystemInterface
 
     bool Set_Time_Of_Day(TimeOfDayType time);
 
+    const char *Get_Path_User_Data() const { return m_userDataDirectory.Str(); }
```

Comment:
Seen in World Builder

Reply:
Should actually return AsciiString instance

Reply:
Very well.